std/bigint.d

```diff
@@ -1025,6 +1025,220 @@ public:
         assert(aa[BigInt(456)] == "def");
     }
 
+    /*
+        Increments an arbitrary-length integer, represented as an array of
+        bytes, using the native endianness.
+
+        This is just used by `toBytes` to convert unsigned bytes of the
+        BigInt to two's complement form.
+    */
+    private static void incrementBytes (ref ubyte[] value) nothrow pure @safe
```

Comment:
Should take an output range of ubyte (confer with other comment proposing to use an Appender)

Reply:
Can you clarify? What should the signature of this method be?

Reply:
a template constraint taking only an output range of ubyte

Reply:
So something like `private static void incrementBytes(Range)(Range result, ubyte[] value) if (isOutputRange!Range)`? You still need to pass the original data in there somehow.

Reply:
Ok. Let's forget the outrange thing in this review and concentrate about the style.